### DISCUSSION TITLE
Database resetted in Heroku

### DISCUSSION THREAD

**Discussion by indiemakersid** [pocketbase/pocketbase#484] [Q&A]
Hey folks,
I am super excited with this pocketbase.
But unfortunately the Database keep resetted after 30 minutes.

Is it because of Free tier ? 
If i use the premium one that never sleep, would the database not resetted ?

Many Thanks

**Comment by Nikhil1920** [comment]
Heroku's filesystem is not persistent even if you use premium the filesystem will reset after a build is run and you will loose your data so it is better not to use heroku for sqlite based application

you can look for other alternatives where the filesystem is persistent like fly.io or if can can manage it better go for a vm like aws lightsail or digital ocean

**Reply by indiemakersid** [reply]
Hey Thanks for quick answer! Did you have any digital ocean deployment guide for pocketbase ?

Many Thanks!

**Reply by indiemakersid** [reply]
I keep on getting this.

<img width="531" alt="image" src="https://user-images.githubusercontent.com/100898798/190555528-7e4c2d6d-7a2c-46db-aeb2-0e1ad4407a15.png">

**Reply by Nikhil1920** [reply]
you need to give pocketbase permission to run. you can do so with the following command `chmod +x /path/to/pocketbase`

**Comment by ganigeorgiev** [comment]
@indiemakersid Just to let you know that I've just published a quick starting guide how to deploy for free on Fly.io - https://github.com/pocketbase/pocketbase/discussions/537
